src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,50 @@
 
 class Animal {
   // write your code here
+  // static alive = [];
+
+  constructor() {
+    this.health = 100;
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
   // write your code here
+  constructor() {
+    super();
+    this.hidden = false;
+
+    Animal.alive.push(this);
+  };
+
+  hide() {
+    this.hidden = true;
+  };
 }
 
 class Carnivore extends Animal {
   // write your code here
+  constructor() {
+    super();
+
+    Animal.alive.push(this);
+  };
+
+  bite(name) {
```

Comment:
`name` intuitively refers to a string, let's call it `victim`
```suggestion
  bite(victim) {
```